### PR TITLE
Ncnews 10

### PR DESCRIPTION
**Pull request by Amir2341**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -216,3 +216,103 @@ describe("GET /api/articles", () => {
...
+      });
+  });
+  test("status 400 for bad comment", () => {
+    return request(app)
+      .post("/api/articles/1/comments")
+      .expect(400)
+      .send({ username: "butter_bridge" })
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
```
Comment (HeatherEver) [inline __tests__/app.test.js:316]: can you please add a test for is the username does not exist please

**Review by HeatherEver — COMMENTED** [review]
Jsut one test to add please

File: __tests__/app.test.js [hunk 2]
```diff
@@ -216,3 +216,112 @@ describe("GET /api/articles", () => {
...
+      .expect(201)
+      .send({
+        username: "butter_bridge",
+        body: "hi",
+      })
+      .then(({ body: { comment } }) => {
+        expect(comment).toEqual(
+          expect.objectContaining({
+            comment_id: expect.any(Number),
+            article_id: expect.any(Number),
```
Comment (karlaferry) [inline __tests__/app.test.js]: article_id shouldn't be any number, should be 3 in this case

File: __tests__/app.test.js [hunk 3]
```diff
@@ -216,3 +216,112 @@ describe("GET /api/articles", () => {
...
+            body: expect.any(String),
+            author: expect.any(String),
```
Comment (karlaferry) [inline __tests__/app.test.js]: author should be `butter_bridge`, you can hardcode stuff that you are 100 percent sure of the value, in this case the comment body and the article_id

**Review by karlaferry — APPROVED** [review]

**Review by karlaferry — APPROVED** [review]

**Pull request merged**